### PR TITLE
Mobile Menu

### PR DESCRIPTION
**Pull request by mwanawabangona**
- Add Html file
- Add js file
- Add CSS file
- Add Github folder in which includes linters.yml file
- Add gitignore file in which includes node-module/ and Vscode files
- Add Linters config files

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,317 +1,375 @@
...
   <head>
-    <meta charset="utf-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <script async src="https://www.googletagmanager.com/gtag/js?id=G-WXBV13V4WN"></script>
+    <script>
+    window.dataLayer = window.dataLayer || [];
+      function gtag(){window.dataLayer.push(arguments);}
+      gtag('js', new Date());
+      gtag('config', 'G-WXBV13V4WN');
+    </script>
```
Comment (rloterh) [inline index.html:10]: - [ ] OPTIONAL
Let's keep our _html_ code clean by keeping **all** JavaScript code in a separate file

File: index.html [hunk 2]
```diff
@@ -1,317 +1,375 @@
...
-
-    <!-- Global site tag (gtag.js) - Google Analytics -->
-  <script async src="https://www.googletagmanager.com/gtag/js?id=GA_MEASUREMENT_ID"></script>
-  <script>
-
-<script>
-  window.dataLayer = window.dataLayer || [];
-  function gtag(){dataLayer.push(arguments);}
-  gtag('js', new Date());
+  <div id="menuId" class="myMenu">
```
Comment (rloterh) [inline index.html:20]: - [ ] Your mobile menu working correctly however, we actually want your design to closely match the provided mobile menu [Figma template](https://www.figma.com/proto/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A546&scaling=min-zoom&page-id=0%3A1). Your font sizes and spacing (margins/paddings) do not match the provided design. Kindly ensure to improve upon the mentioned aspect in order to enhance the UI of your mobile menu

**Your design (mobile menu):**

image


**The original design (mobile menu):**

image

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Ellis, 

You did extremely great on this project!👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there!👍 

## Good Points 👍  
- Highly organized/clean code
- Good ReadMe
- No linter errors


## Required Changes ♻️
- [ ] _Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [ ] _Kindly consider  the comments under the review marked [OPTIONAL]._


Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mikenath223 — APPROVED** [review]
Hello there  

Your project is complete!  There is nothing else to say other than... it's time to merge it 🛳 
Congratulations! 🎉 

Happy coding. Cheers! 🚀🚀

If you feel something isn't clear enough you can leave a question or comment in the PR thread.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**